### DISCUSSION TITLE
In vivaldi browser notes showing as on phone

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3679] [Q&A] [answered]
Hey,

I change browser to vivaldi browser (they are on chrome engine if matter), and since them trilium sometimes run as on phone. I know that I can change it manually and do, but after sometime problem came back.

Am I able to workaround somehow this problem from your perspective?
[There are](https://i.imgur.com/mnyDttO.png) my cookies, LS is empty

Thanks from advance!
Simon

**Comment by zadam** [comment, marked as answer]
Hello, I changed the mobile detection algorithm to be more precise, this should help.

**Reply by shirobachi** [reply]
Thanks a lot!

**Reply by shirobachi** [reply]
Hey, 
I am afraid it did not helped ;/ I am on version 0.59.1

**Reply by zadam** [reply]
The fix was not released yet.

**Reply by shirobachi** [reply]
Oh, I got it. Sorry I misunderstood situation. Thanks for clarification :)
